### ISSUE TITLE
[webpack] not resolving imports with resource query

### ISSUE THREAD

**Issue by daltones** [benmosher/eslint-plugin-import#357]
There are not many loaders that use this feature, but Webpack provides the *resource* query besides the *loader* query (inside the loader context: `this.resourceQuery` besides `this.query`). Like this:

```js
// common loader query
import Foo from 'some-loader?param=value!module-path.js';

// loader query and resource query
import Bar from 'some-loader?param=value!module-path.js?otherParam=otherValue';
```

Currently Webpack Resolver doesn't understand resource queries and cannot resolve `module-path.js` properly.

**Comment by benmosher** [comment]
Ah, yep. It knows to omit the loader bits, but I don't recall handling the query. 

Shouldn't be too tough to add in.

**Comment by benmosher** [comment]
FYI, just (finally!) published with v0.3.2 of the resolver.

**Comment by daltones** [comment]
Good! Thanks =)
